marketplace/core/types/channels/facebook/views.py

```diff
@@ -20,10 +22,30 @@ def configure(self, request, **kwargs):
         """
         Adds a config on specified App and create a channel on weni-flows
         """
+        app = self.get_object()
         self.serializer_class = FacebookConfigureSerializer
-        serializer = self.get_serializer(self.get_object(), data=request.data)
+        serializer = self.get_serializer(app, data=request.data)
         serializer.is_valid(raise_exception=True)
 
         self.perform_update(serializer)
 
+        user = request.user
+        client = ConnectProjectClient()
```

Comment:
A small suggestion would be to migrate this instance to line 43, because in this current way, if the app has already been configured you are instantiating the client and not using it

Reply:
Done